src/frontend/src/expr/function_call.rs

```diff
@@ -111,19 +111,19 @@ impl FunctionCall {
 
     /// Create a cast expr over `child` to `target` type in `allows` context.
```

Comment:
Should we note somewhere that developer need to ensure not to touch `child` if to return an error?